docs/development/update-helm-charts.md

```diff
@@ -0,0 +1,42 @@
+# Update helm charts for EKS-A Packages
+
+Helm charts used in EKS-A Packages are required to go through a series of modification to meet release standards. This file documents some standard procedures and modifications that are required for helm charts used in EKS-A Packages.
+
+## Generate patch files
+
+Helm charts modifications are done through patches. To do so, we perform the following procedures:
+- clone the target helm chart repo locally;
+- update helm charts locally (see details at next step); and 
+- generate patch files using [`git format-patch`](https://git-scm.com/docs/git-format-patch).
+
+## Update helm charts locally
+### Update `values.yaml`
+
+Following changes need to be made to the `values.yaml` file:
+
+| Field                                   | Action            | Value                                     |
+| :-------------------------------------: | :-------------:   | :----------------------------------------:|
+| `sourceRegistry`                        | add               | `public.ecr.aws/eks-anywhere`             |
+| `image:repository`                      | modify            | $(HELM_DESTINATION_REPOSITORY)[^1]        |
+| `image:tag`                             | delete            | N/A       
+| `image:digest`                          | add               | `{{`$(HELM_DESTINATION_REPOSITORY)[^1]`}}`|
+
+### Update `templates` directory
+Following changes need to be made to the yaml files under `templates`:
+
+| Field                                   | Action            |  Value                                    |
+| :-------------------------------------: | :---------------: | :---------------------------------------: |
+| `metadata:namespace`[^2]                | add               | `{{ .Release.Namespace \| quote }}`       |
+| `spec:template:spec:containers: image`  | modify            | `{{ .Values.sourceRegistry }}/{{ .Values.controller.image.repository }}@{{ .Values.controller.image.digest }}`        |
+
```

Comment:
Along with the previous pullSecrets in values added

| `spec:template:spec:` | add | `    {{- with .Values.imagePullSecrets }} / imagePullSecrets: / {{- toYaml . | nindent 8 }} / {{- end }}` |

Reply:
Added